Gemfile

```diff
@@ -19,7 +19,17 @@ gem 'newrelic_rpm'
 gem 'pg'
 gem 'puma'
 gem 'rails'
+gem 'rake'
```

Comment:
minor comment for this project as it's the pet project but in general it's always a good idea to add the gem version beside it's name to prevent issue in the future in case you want to upgrade/downgrade gem versions. Or if you're gemfile.lock gets deleted by mistake.